server/routes/api.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const router = express.Router()
+const APImanager = require('./APImanager')
+const apiManager = new APImanager
+
+setInterval(() => {
+    apiManager.getPlayers()
+    }, 1000*60*60*24);
+    
+    apiManager.getPlayers()
+
+    
+
+    router.get('/teams/:teamName',function (req,res) {
+        const teamName=req.params.teamName
```

Comment:
you can deconstruct objects. it looks a lot better
`const { teamName } = req.params`

it is the same as
`const teamName = req.params.teamName`